src/Compute/Compute/Generated/VirtualMachineScaleSet/Config/NewAzureRmVmssConfigCommand.cs

```diff
@@ -347,6 +347,16 @@ public partial class NewAzureRmVmssConfigCommand : Microsoft.Azure.Commands.Reso
            Mandatory = false)]
         public bool? EnableSecureBoot { get; set; } = null;
 
+        [Parameter(
+            Mandatory = false,
+            HelpMessage = "Enable resilient VM creation.")]
+        public SwitchParameter EnableResilientVMCreate { get; set; }
+
+        [Parameter(
+            Mandatory = false,
+            HelpMessage = "Enable or disable resilient VM deletion.")]
+        public bool EnableResilientVMDelete { get; set; }
```

Comment:
why is the other one Switch  and this one bool ?

Reply:
Addressed with design author and changed, Switch is for New-Azvmssconfig and Update is for Update-Azvmss